react-native-purchases-ui/android/src/main/java/com/revenuecat/purchases/react/ui/FontAssetManager.kt

```diff
@@ -0,0 +1,53 @@
+package com.revenuecat.purchases.react.ui
+
+import android.content.res.AssetManager
+import androidx.compose.ui.text.font.Font
+import androidx.compose.ui.text.font.FontStyle
+import androidx.compose.ui.text.font.FontWeight
+import com.facebook.infer.annotation.Nullsafe
+
+@Nullsafe(Nullsafe.Mode.LOCAL)
+object FontAssetManager {
+    private val mFontCache: MutableMap<String?, List<Font>?> = hashMapOf()
+    private val EXTENSIONS = arrayOf("", "_bold", "_italic", "_bold_italic")
+    private val FILE_EXTENSIONS = arrayOf(".ttf", ".otf")
+
+    private fun getStyleFromExtension(style: Int): FontStyle {
+        return when (style) {
+            2, 3 -> FontStyle.Italic
+            else -> FontStyle.Normal
+        }
+    }
+
+    private fun getWeightFromExtension(style: Int): FontWeight {
+        return when (style) {
+            1, 3 -> FontWeight.Bold
+            else -> FontWeight.Normal
+        }
+    }
+
+    fun getFontList(
```

Comment:
NABD, but this could return the `FontFamily` directly, instead of creating it on the caller? IMO it might be slighty cleaner

Reply:
sure! i initially thought of returning list just incase we need to reuse it for some reason, but I don't think we will. I'll change it to return font family